python/cudf/tests/test_groupby.py

```diff
@@ -35,16 +37,47 @@ def get_nelem():
         yield elem
 
 
+@pytest.fixture
+def gdf():
+    return DataFrame({'x': [1, 2], 'y': [1, 2]})
+
+
+@pytest.fixture
+def pdf(gdf):
+    return gdf.to_pandas()
+
+
+@pytest.mark.parametrize('as_index', [True, False])
+def test_groupby_no_index(pdf, gdf, as_index):
```

Comment:
This test name is confusing because `index` is ambiguous here. I believe you're referring to the `__getitem__` behavior as opposed to a dataframe index.

Reply:
`as_index` in the context of groupby means: convert whatever columns are used to groupby into the dataframe index. When `as_index=False`, the groupby columns are left in the dataframe and the index is untouched. I understand the ambiguity, I think, but I'm not sure what a better naming scheme would be. I'm open to suggestions.

Reply:
I parametrized and didn't change the name. I'll do that and resolve the other as well.